src/Microsoft.Health.Dicom.Functions/Update/Models/CompleteStudyArguments.cs

```diff
@@ -12,14 +15,22 @@ public sealed class CompleteStudyArguments
 {
     public int PartitionKey { get; }
 
+    public IReadOnlyList<InstanceMetadata> InstanceMetadatas { get; }
```

Comment:
nit: "metadatas" bugs me because metadata is already plural lol, but I also understand why the name. Maybe some alternative name?